Front-End/src/Components/Login.js

```diff
@@ -1,15 +1,47 @@
 import React from "react";
-import { Link, useNavigate} from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import "../CSS/Login.css";
 import Gif from "../Resources/Chalk_Shapian.gif";
-let google = "https://cdn1.iconfinder.com/data/icons/google-s-logo/150/Google_Icons-09-512.png";
+let google =
+  "https://cdn1.iconfinder.com/data/icons/google-s-logo/150/Google_Icons-09-512.png";
 
 const Login = () => {
   const navigate = useNavigate();
-  const login = () => {
-    localStorage.setItem('LoggedIn',true);
-    navigate("/Discover");
-  }
+
+  const login = async () => {
+    const email = document.getElementById("emailLogin").value;
+    const password = document.getElementById("passwordLogin").value;
+    const rememberMe = document.getElementById("check").checked;
+  
+    try {
+      const response = await fetch("http://localhost:6969/api/user/login", {
```

Comment:
Add `http://localhost:6969` to `.env` and use here using `process.env`.

Reply:
done